src/stories/50_User_Consent/Story_User_Consent.html

```diff
@@ -0,0 +1,167 @@
+<!--- {
+  "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  This example shows how to integrate `amp-consent` in AMP Stories,
+  to offer users control over data sharing and cookies.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="/stories/user_consent/story_user_consent/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>AMP Story with User Consent</title>
+
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+
+  <!-- ## Setup -->
+  <!-- Import the `amp-story` extension for creating an AMP Story. -->
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+
+  <!-- ... and the `amp-consent` extension. -->
```

Comment:
if you use the ... you can add the same ones at the end of the previous comment

Reply:
done